app/validators/representation_order_validator.rb

```diff
@@ -67,6 +67,7 @@ def validate_maat_reference
     return unless case_type&.requires_maat_reference?
     validate_presence(:maat_reference, 'invalid')
     validate_pattern(:maat_reference, /^[0-9]{7,10}$/, 'invalid')
+    matt_reference_uniqueness(:maat_reference, 'unique')
```

Comment:
long winded but i would use `validate_maat_reference_uniqueness` for consistency and clarity